src/main/java/org/kohsuke/github/GHExternalGroups.java

```diff
@@ -0,0 +1,28 @@
+package org.kohsuke.github;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * A list of external groups.
+ *
+ * @author Miguel Esteban Gutiérrez
+ */
+public class GHExternalGroups {
```

Comment:
Modify this to match 
https://github.com/hub4j/github-api/blob/main/src/main/java/org/kohsuke/github/GHCommitFilesPage.java

Rename to `GHExternalGroupsPage`
Make `groups` an array.

Reply:
Done. I don't like handling arrays, but understand that is a particular pattern used across the client.

Reply:
What your change shows is that maybe we don't need to use arrays.  Could you file an issue for this?